### ISSUE TITLE
[Feature Request] Add "-w 0" to the Windows Terminal default settings

### ISSUE THREAD

**Issue by PConrad-NI** [ForkIssues/TrackerWin#2368]
Currently the Windows Terminal default settings are just `-d .` which opens a Windows Terminal in the current directory. It however will spawn an additioal Windows Terminal session which defeats the purpose of using Windows Terminal.

If `-w 0` is added (`-w 0 -d `), the newly created terminal panes will be created in the last used Windows Terminal session rather than spawning a new session.

**Comment by DanPristupov** [comment]
Not sure about this one.

> Currently the Windows Terminal. .. opens a Windows Terminal in the current directory. It however will spawn an additional Windows Terminal session.

Do you mean it opens a new terminal window instead of new tab? This is the same behavior for all terminals on both Mac and Windows. I don't want to change behavior for a particular terminal.

**Comment by PConrad-NI** [comment]
Yes, current behavior opens a whole new terminal window rather than adding a tab to an existing window. That is the purpose of changing from discrete powershell or cmd or bash to WindowsTerminal. If someone wants multiple windows spawned, sticking with the other options is still valid and will spawn multiple windows.

![image](https://github.com/user-attachments/assets/4883ac3b-dadf-4d88-b0b5-e4f6d38fd076)

**Comment by jerone** [comment]
At least making the "Arguments" input editable.

**Comment by DanPristupov** [comment]
@jerone you can select 'Custom' shell and configure arguments manually.

**Comment by jerone** [comment]
Yeah, I can ofcourse copy the path from WindowsTerminal.
